src/systems/systems/src/pressurization/mod.rs

```diff
@@ -106,6 +224,114 @@ impl Default for Pressurization {
     }
 }
 
+pub struct PressurizationOverheadPanel {
+    mode_sel: AutoManFaultPushButton,
+    man_vs_ctl_switch: SpringLoadedSwitch,
+    ldg_elev_knob: ValueKnob,
+    ditching: NormalOnPushButton,
+}
+
+impl PressurizationOverheadPanel {
+    pub fn new() -> Self {
+        Self {
+            mode_sel: AutoManFaultPushButton::new_auto("PRESS_MODE_SEL"),
+            man_vs_ctl_switch: SpringLoadedSwitch::new("PRESS_MAN_VS_CTL"),
+            ldg_elev_knob: ValueKnob::new_with_value("PRESS_LDG_ELEV", -2000.),
+            ditching: NormalOnPushButton::new_normal("PRESS_DITCHING"),
+        }
+    }
+
+    fn ldg_elev_is_auto(&self) -> bool {
+        let margin = 100.;
+        (self.ldg_elev_knob.value() + 2000.).abs() < margin
+    }
+
+    fn is_in_man_mode(&self) -> bool {
+        !self.mode_sel.is_auto()
+    }
+
+    fn man_vs_switch_position(&self) -> usize {
+        self.man_vs_ctl_switch.toggle_position()
+    }
```

Comment:
**thought**: Different names to describe the same thing. `toggle_position` makes me think the position will be changed (it feels like an action to me). Should `SpringLoadedSwitch.toggle_position` maybe be renamed to `SpringLoadedSwitch.position` or `SpringLoadedSwitch.switch_position` (I like the first better as `Switch` is already part of the type name).

Reply:
Changed name to `position`.